### PR TITLE
Add copy of text option

### PR DESCRIPTION
**Pull request by panam1916**
Issue #110 

Add Feature of Copy Text option

### CODE REVIEW

File: Frontend/src/pages/message/message.js [hunk 1]
```diff
@@ -59,7 +51,7 @@ function Message() {
   useEffect(() => {
     if (chatId !== null) {
       //make a websocket connection here
-      let wss = new WebSocket(`wss://mindmate-ws.onrender.com?id=${chatId}`);
+      let wss = new WebSocket(`ws://mindmate-ws.onrender.com?id=${chatId}`);
```
Comment (afeefuddin) [inline Frontend/src/pages/message/message.js:54]: Why have you changed this?

File: Frontend/src/pages/message/Chat.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useState } from 'react';
+import Markdown from 'react-markdown';
+import { LuCopy } from 'react-icons/lu'; // Assuming LuCopy is an icon for copy
```
Comment (afeefuddin) [inline Frontend/src/pages/message/Chat.js:4]: No need to add a comment here

**Review by afeefuddin — CHANGES_REQUESTED** [review]

File: Frontend/src/pages/message/message.js (hunk 1 above)
Comment (panam1916) [reply Frontend/src/pages/message/message.js:54]: oh sorry i just change and push it again

**Comment by panam1916** [comment]
Hey just check is it alright?

**Comment by afeefuddin** [comment]
Squash both the commits

**Comment by panam1916** [comment]
> Squash both the commits

Hey , I squash both commits Please Check it


 ([detached HEAD 2aa3343] Add copy button to text and other changes
 Date: Wed May 22 18:24:54 2024 +0530
 3 files changed, 54 insertions(+), 9 deletions(-)
 create mode 100644 Frontend/src/pages/message/Chat.js
Successfully rebased and updated refs/heads/copy.)

**Comment by afeefuddin** [comment]
@panam1916 Now I see 4 commits, Read about squashing [here](https://stackoverflow.com/questions/35703556/what-does-it-mean-to-squash-commits-in-git)

**Comment by panam1916** [comment]
> @panam1916 Now I see 4 commits, Read about squashing [here](https://stackoverflow.com/questions/35703556/what-does-it-mean-to-squash-commits-in-git)

so i remove commits which are not squash?

**Comment by panam1916** [comment]
Do I need to update my branch?

**Comment by panam1916** [comment]
> @panam1916 Now I see 4 commits, Read about squashing [here](https://stackoverflow.com/questions/35703556/what-does-it-mean-to-squash-commits-in-git)

Hi Can you Check and tell me whether there is any mistake or not

**Comment by panam1916** [comment]
Hey there happen lot of confusion please guide me what i do

**Comment by panam1916** [comment]
> @panam1916 Now I see 4 commits, Read about squashing [here](https://stackoverflow.com/questions/35703556/what-does-it-mean-to-squash-commits-in-git)

Hey I created New pr #139 Can you check that in this already lot of messup

**Pull request closed without merging**